scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,24 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
+  while read -r line; do
+    comment=$(echo "$line" | cut -c1)
+    if [ ! "$comment" = "#" ]; then               # lines starting with # get ignored
+      owner=$(echo "$line" | awk '{ print $1 }')
+      perms=$(echo "$line" | awk '{ print $2 }')
+      file=$(echo  "$line" | awk '{ print $3 }')
+      mkdir -p ../rootfs"$(dirname "$file")"
+      cp -v config/files"$file" ../rootfs"$file"
```

Comment:
could you add a check if "config/files${file}" is a directory?
that way we'd be able to create and (even more important) set permissions on directories as well...

all else should stay the same, just skip the copy part if "config/files${file}" is a directory :thought_balloon: